frameworks/cassandra/tests/test_zzzrecovery.py

```diff
@@ -66,9 +66,12 @@ def test_node_replace_replaces_node():
 # @@@@@@@
 @pytest.mark.sanity
 def test_shutdown_host():
-    replace_task = sdk_tasks.get_task_avoiding_scheduler(
+    candidate_tasks = sdk_tasks.get_tasks_avoiding_scheduler(
         config.SERVICE_NAME, re.compile('^node-[0-9]+-server$'))
-    assert replace_task is not None, 'Could not find a node to shut down'
+    assert len(candidate_tasks) != 0, 'Could not find a node to shut down'
```

Comment:
if we should never have two pods on the same agent, should we:
```
assert len(candidate_tasks) == 1
```
?

Reply:
This is the list of tasks which aren't colocated with the scheduler. Would expect >=2.

Reply:
ahhh, my mistake